From 99b75144c6a58d521984649058c6b75221ff5859 Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Mon, 18 Dec 2017 08:36:10 +0200
Subject: [PATCH 3127/5242] MLK-17232-3: dts: arm64: imx8mq: remove APB clock
 for rm67191 DTS

commit  989ae959546abbeedf753cd46aa5bd4c6425ba82 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to commit:

<8c9aa9e83e37> "MLK-16942-2: dts: Remove disp_apb clock rate setting"

APB clock is configured by ROM code and it should be 133MHz. Remove the
setting from this DTS.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts  |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
index f4490d8..804a6c8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
@@ -27,20 +27,17 @@
 	disp-dev = "mipi_disp";
 
 	assigned-clocks = <&clk IMX8MQ_CLK_DISP_AXI_SRC>,
-			  <&clk IMX8MQ_CLK_DISP_APB_SRC>,
 			  <&clk IMX8MQ_CLK_DISP_RTRM_SRC>,
 			  <&clk IMX8MQ_CLK_DISP_RTRM_PRE_DIV>,
 			  <&clk IMX8MQ_CLK_DC_PIXEL_SRC>,
 			  <&clk IMX8MQ_VIDEO_PLL1_REF_SEL>,
 			  <&clk IMX8MQ_VIDEO_PLL1>;
 	assigned-clock-parents = <&clk IMX8MQ_SYS1_PLL_800M>,
-				 <&clk IMX8MQ_SYS2_PLL_125M>,
 				 <&clk IMX8MQ_SYS1_PLL_800M>,
 				 <&clk IMX8MQ_CLK_DISP_RTRM_CG>,
 				 <&clk IMX8MQ_VIDEO_PLL1_OUT>,
 				 <&clk IMX8MQ_CLK_25M>;
 	assigned-clock-rates = <800000000>,
-			       <125000000>,
 			       <400000000>,
 			       <400000000>,
 			       <120000000>,
-- 
1.7.9.5

